packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -155,3 +158,119 @@ export function getNetworkName(
       return 'Unknown'
   }
 }
+
+const SEVEN_DAYS_IN_SECONDS = 7 * 24 * 60 * 60
+
+const mainnetTokenBridge = {
+  l1GatewayRouter: '0x72Ce9c846789fdB6fC1f34aC4AD25Dd9ef7031ef',
+  l2GatewayRouter: '0x5288c571Fd7aD117beA99bF60FE0846C4E84F933',
+  l1ERC20Gateway: '0xa3A7B6F88361F48403514059F1F16C8E78d60EeC',
+  l2ERC20Gateway: '0x09e9222E96E7B4AE2a407B98d48e330053351EEe',
+  l1CustomGateway: '0xcEe284F754E854890e311e3280b767F80797180d',
+  l2CustomGateway: '0x096760F208390250649E3e8763348E783AEF5562',
+  l1WethGateway: '0xd92023E9d9911199a6711321D1277285e6d4e2db',
+  l2WethGateway: '0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B',
+  l2Weth: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
+  l1Weth: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+  l1ProxyAdmin: '0x9aD46fac0Cf7f790E5be05A0F15223935A0c0aDa',
+  l2ProxyAdmin: '0xd570aCE65C43af47101fC6250FD6fC63D1c22a86',
+  l1MultiCall: '0x5ba1e12693dc8f9c48aad8770482f4739beed696',
+  l2Multicall: '0x842eC2c7D803033Edf55E478F461FC547Bc54EB2'
+}
+
+const mainnetETHBridge = {
+  bridge: '0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a',
+  inbox: '0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f',
+  sequencerInbox: '0x1c479675ad559DC151F6Ec7ed3FbF8ceE79582B6',
+  outbox: '0x0B9857ae2D4A3DBe74ffE1d7DF045bb7F96E4840',
+  rollup: '0x5eF0D09d1E6204141B4d37530808eD19f60FBa35',
+  classicOutboxes: {
+    '0x667e23ABd27E623c11d4CC00ca3EC4d0bD63337a': 0,
+    '0x760723CD2e632826c38Fef8CD438A4CC7E7E1A40': 30
+  }
+}
+
+export const networksList: { [chainId: number]: L1Network | L2Network } = {
+  [ChainId.ArbitrumOne]: {
+    chainID: 42161,
```

Comment:
they are exported from the sdk so you should be able to use them
```
import {
  l1Networks,
  l2Networks
} from '@arbitrum/sdk/dist/lib/dataEntities/networks'

export const networksList: { [chainId: number]: L1Network | L2Network } = {
  ...l1Networks,
  ...l2Networks
}
```

we probably want to filter out the networks we don't need for the network switch UI

Reply:
They are exported internally in SDK. But not through the SDK interface to the whole world.
Anyway, refactored this completely and not hardcoding all network details in UI now :) .